web/src/components/Testimonial/Testimonial.tsx

```diff
@@ -43,7 +43,8 @@ export const Testimonial: React.FC<TestimonialProps> = (
 		{
 			[styles[`testimonial--${kebabCaseVariantClassName}`]]: variant,
 		},
-		className
+		className,
+		"testimonial-max-width"
```

Comment:
Can we just use 'testimonial' as this is a class name for the testimonial 'block'? Think max-width is confusing when we're applying it globally and we already have the `full-width` variants